app/layout/community/component.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+
+import Button from 'components/button';
+import Column from 'layout/statics/column';
+import Wrapper from 'layout/wrapper';
+
+export interface CommunityInfoProps {
+
+}
+
+export const CommunityInfo: React.FC<CommunityInfoProps> = () => {
+  return (
+    <Wrapper>
+      <div className="grid w-full max-w-5xl grid-cols-2 py-16 mx-auto border-t border-white gap-x-36 border-opacity-20">
+        <Column
+          title="Explore the projects shared by the community"
+          subtitle="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore."
+          description="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor."
+          caption="Explore projects"
+          href="community/projects"
```

Comment:
use absolute paths. is this an internal link? I see you have a prop for setting this